### ISSUE TITLE
"Chrome is being controlled by automated test software" banner failure

### ISSUE THREAD

**Issue by alarimer** [galenframework/galen#557]
Java: 8 Update 161
GalenFramework-CLI 2.3.6
Chrome 64.0.3282.167

"Chrome is being controlled by automated test software” banner causes GSpec failure for "centered vertically inside screen" but not for headless.

![galen chrome](https://user-images.githubusercontent.com/17734739/36328668-064be52c-1331-11e8-9c11-b38ffff80e45.png)

see also related but not mine: https://sqa.stackexchange.com/q/26051

**Comment by diemol** [comment]
That bar can be disabled by using `--disable-infobars`.

https://sqa.stackexchange.com/questions/26051/chrome-driver-2-28-chrome-is-being-controlled-by-automated-test-software-noti

**Comment by NasserMahmoud1** [comment]
I have the same issue, but since --disable-infobars has been removed according to this "https://chromium.googlesource.com/chromium/src/+/d869ab3350d8ebd95222b4a47adf87ce3d3214b1" 
And i still searching for replacement

**Comment by diemol** [comment]
Yeah, my previos comment is not valid anymore since Chrome 64(65?), it was removed by the Chrome team.

**Comment by hypery2k** [comment]
the PR should still remove it, as it uses another still valid flag ;)
